migrations/sql/V2018.10.30.1809__UpdateRolesAndPermissions.sql

```diff
@@ -0,0 +1,29 @@
+DELETE FROM role_permission
+WHERE role = 1
+AND permission IN ('estimates.read',
+                           'modelling-groups.read',
+                           'users.read');
+
+
+DO $$
+  declare admin int;
+  declare developer int;
+  declare member int;
+begin
+  SELECT id INTO admin FROM role WHERE name = 'admin';
+  SELECT id INTO developer FROM role WHERE name = 'developer';
+  SELECT id INTO member FROM role WHERE name = 'member';
```

Comment:
does the `modelling-group` scope here interact with this at all? I must admit I've fallen behind on understanding the auth model and if you go through it with Emma I'd appreciate a refresher